From 7f10d411b99d68c7a954030d67f3693e964b51fe Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Tue, 12 Dec 2023 10:38:32 +0000
Subject: [PATCH] Build external angle by python3

This patch will enable to build generate_registry_tables.py
file to use python3 instead of python2
This patch needs to be reverted when we make python3
as default.

Tests-Done: Build and boot the device.

Tracked-On: OAM-114192
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 .../spirv-tools/src/utils/generate_registry_tables.py           | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/vulkan-deps/spirv-tools/src/utils/generate_registry_tables.py b/third_party/vulkan-deps/spirv-tools/src/utils/generate_registry_tables.py
index 14d4909fa6..69628faada 100755
--- a/third_party/vulkan-deps/spirv-tools/src/utils/generate_registry_tables.py
+++ b/third_party/vulkan-deps/spirv-tools/src/utils/generate_registry_tables.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # Copyright (c) 2016 Google Inc.
 
 # Licensed under the Apache License, Version 2.0 (the "License");
-- 
2.39.2

